gcc/rust/resolve/rust-late-name-resolver-2.0.cc

```diff
@@ -254,6 +254,8 @@ Late::visit (AST::PathInExpression &expr)
   // TODO: How do we have a nice error with `can't capture dynamic environment
   // in a function item` error here?
   // do we emit it in `get<Namespace::Labels>`?
+  if (expr.is_lang_item ())
+    return;
```

Comment:
I think resolution here should be pretty easy but we'll do this in another PR - I'm happy with that fix